src/static/doctor.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+
+<head>
+  <meta charset="utf-8">
+  <style type="text/css">
+    table {
+      font-family: "Arimo", "Surt", "Helvetica", "Lucida Sans", "Lucida Sans Unicode", "Luxi Sans", Tahoma, sans-serif;
+      box-shadow: 1px 1px 10px rgba(0, 0, 0, 0.5);
+      border-collapse: collapse;
+      border-spacing: 0;
+    }
+
+    table {
+      margin: auto;
+    }
+
+    table,
+    td,
+    th {
+      font-size: 14px;
+      padding: 4px;
+      text-align: center;
+      border: 1px solid #225560;
+      color: #1A1A1A;
+    }
+
+    th {
+      background-color: #225560;
+      color: white !important;
+      font-size: 14px;
+    }
+
+  </style>
+</head>
+
+<body>
+  <script src="https://d3js.org/d3.v3.js"></script>​
+  <script>
+    const history = %% COMMAND_HISTORY %%;
+
+    d3.select("body").selectAll("table").data([history]).enter().append("table")
+      .call(build_table);
+
+    function build_table(sel) {
+      sel.each(function (data) {
+        data = data instanceof Array ? data : [data];
+
+        let colnames, tds, table = d3.select(this);
+
+        colnames = data.reduce((key_names, col) => key_names.concat(d3.keys(col)), [])
+          .reduce((key_names, col) => (key_names.set(col, 0), key_names), d3.map())
+          .keys();
+
+        colnames?.sort();
+
+        // create header row
+        table.append("thead").append("tr").selectAll("th").data(colnames).enter().append("th")
+          .text((header_text) => header_text);
+
+        // create the table cells
+        tds = table.append("tbody").selectAll("tr").data(data).enter().append("tr").selectAll("td")
+          .data((table_cell) => colnames.map((data_key) => table_cell[data_key] || "")).enter()
+          .append("td");
+
+        // data type if not array
+        tds.filter((cell_contents) => !(cell_contents instanceof Array || cell_contents instanceof Object))
+          .text((cell_contents) => cell_contents);
+
+        // new table if array or object
+        tds.filter((cell_contents) => (cell_contents instanceof Array || cell_contents instanceof Object))
+          .append("table").call(build_table);
+      });
+    }
+  </script>
+  </script>
```

Comment:
Was reading this PR for fun and don't know what most of it does, but it looks like this is a dupe`</script>` tag 😛

Reply:
removed extra <script> tag